packages/wdio-browserstack-service/src/exitHandler.ts

```diff
@@ -9,10 +10,26 @@ import { BStackLogger } from './bstackLogger.js'
 const __filename = fileURLToPath(import.meta.url)
 const __dirname = path.dirname(__filename)
 
+function getSignalName(code: number) : string|null {
+    if (!code) {
+        return null
+    }
+    if (code > 128) {
```

Comment:
shouldn't it be `code >= 128`, else when code is 128, we may not have exitStatus 0